<p class="2022">Shalev TJ, El-Dien OG, Yuen MMS, Shengqiang S, Jackman SD, Warren RL, Coombe L, van der Merwe L, Stewart A, Boston LB, Plott C, Jenkins J, He G, Yan J, Yan M, Guo J, Breinholt JW, Neves LG, Grimwood J, Rieseberg LH, Schmutz J, Birol I, Kirst M, Yanchuk AD, Ritland C, Russell JH, Bohlmann J. 2022. The western redcedar genome reveals low genetic diversity in a self-compatible conifer. <a href=https://doi.org/10.1101/gr.276358.121>Genome Res. gr.276358.121 Published in Advance September 15, 2022, doi:10.1101/gr.276358.121</a></p>

<p class="2022">Kazemi P, Wong J, Nikolic V, Mohamadi H, Warren RL, Birol I. 2022. ntHash2: recursive spaced seed hashing for nucleotide sequences.<a href=https://doi.org/10.1093/bioinformatics/btac564>Bioinformatics. https://doi.org/10.1093/bioinformatics/btac564</a></p>

<p class="2022">Warren RL. 2022. PASS: De novo assembler for short peptide sequences. <a href=https://arxiv.org/pdf/2208.05598.pdf>arXiv. https://doi.org/10.48550/arXiv.2208.05598</a></p>

<p class="2022">Lin D, Sutherland D, Aninta SI, Louie N, Nip KM, Li C, Yanai A, Coombe L, Warren RL, Helbing C, Hoang LMN, Birol I. 2022. Mining Amphibian and Insect Transcriptomes for Antimicrobial Peptide Sequences with rAMPage. <a href=https://doi.org/10.3390/antibiotics11070952>Antibiotics. 11:952. https://doi.org/10.3390/antibiotics11070952</a></p>

<p class="2022">Gagalova KK, Warren RL, Coombe L, Wong J, Nip KM, Yuen MMS, Whitehill GA, Celedon JM, Ritland C, Taylor GA, Cheng D, Plettner P, Hammond SA, Mohamadi H, Zhao Y, Moore RA, Mungall AJ, Boyle B, Laroche J, Cottrell J, Mackay JJ, Lamothe M, Gérardi S, Isabel N, Pavy N, Jones SJM, Bohlmann J, Bousquet J, Birol I. 2022. Spruce giga-genomes: structurally similar yet distinctive with differentially expanding gene families and rapidly evolving genes. <a href=https://doi.org/10.1111/tpj.15889>The Plant Journal. https://doi.org/10.1111/tpj.15889</a></p>

<p class="2022">Nikolic V, Afshinfard A, Chu J, Wong J, Coombe L, Nip KM, Warren RL, Birol I. 2022. RResolver: efficient short-read repeat resolution within ABySS. <a href=https://doi.org/10.1186/s12859-022-04790-z>BMC bioinformatics. 23:246. https://doi.org/10.1186/s12859-022-04790-z</a></p>

<p class="2022">Wright K, Nip KM, Kim JE, Birol, I. 2022. Seasonal and sex-dependent gene expression in emu (Dromaius novaehollandiae) fat tissues. <a href=https://doi.org/10.1038/s41598-022-13681-5>Sci Rep 12:9419. https://doi.org/10.1038/s41598-022-13681-5</a></p>

<p class="2022">Afshinfard A, Jackman SD, Wong J, Coombe L, Chu J, Nikolic V, Dilek G, Malkoç Y, Warren RL, Birol I. 2022. Physlr: Next-Generation Physical Maps. <a href=https://doi.org/10.3390/dna2020009>DNA 2:116-130. https://doi.org/10.3390/dna2020009</a></p>

<p class="2022">Chiu R, Rajan-Babu IS, Birol I, Friedman JM. 2022. Linked-read sequencing for detecting short tandem repeat expansions. <a href=https://doi.org/10.1038/s41598-022-13024-4>Sci Rep 12:9352. https://doi.org/10.1038/s41598-022-13024-4</a></p>

<p class="2022">Yoo S, Garg E, Elliott LT, Hung RJ, Halevy AR, Brooks JD, Bull SB, Gagnon F, Greenwood CMT, Lawless JF, Paterson AD, Sun L, Zawati MH, Lerner-Ellis J, Abraham RJS, Birol I, et al. 2022. HostSeq: A Canadian Whole Genome Sequencing and Clinical Data Resource. <a href=https://doi.org/10.1101/2022.05.06.22274627>medRxiv 2022.05.06.22274627; doi: https://doi.org/10.1101/2022.05.06.22274627</a></p>

<p class="2022">Li JX, Coombe L, Wong J, Birol I, Warren RL. 2022. ntEdit+Sealer: Efficient targeted error resolution and automated finishing of long-read genome assemblies. <a href=https://doi.org/10.1002/cpz1.442>Curr. Protocols. 2:e442 doi: 10.1002/cpz1.442</a></p>

<p class="2022">Gagalova KK, Whitehill JGA, Culibrk L, Lin D, Lévesque-Tremblay V, Keeling CI, Coombe L, Yuen MMS, Birol I, Bohlmann J, Jones SJM. 2022. The genome of the forest insect pest Pissodes strobi reveals genome expansion and evidence of a Wolbachia endosymbiont. <a href=https://doi.org/10.1093/g3journal/jkac038>G3 Genes|Genomes|Genetics. 12(4):jkac038 doi: 10.1093/g3journal/jkac038</a></p>

<p class="2022">Li C, Sutherland D, Hammond SA, Yang C, Taho F, Bergman L, Houston S, Warren RL, Wong T, Hoang LMN, Cameron CE, Helbing CC, Birol I. 2022. AMPlify: attentive deep learning model for discovery of novel antimicrobial peptides effective against WHO priority pathogens. <a href=https://doi.org/10.1186/s12864-022-08310-4>BMC genomics. 23:77. https://doi.org/10.1186/s12864-022-08310-4</a></p>

<p class="2022">Selinger AJ, Cavallin NA, Yanai A, Birol I, Hof F. 2022. Template-Directed Synthesis of Bivalent, Broad-Spectrum Hosts for Neuromuscular Blocking Agents. <a href=https://doi.org/10.1002/anie.202113235>Angew Chem Int Ed Engl. 61(4):e202113235. doi: 10.1002/anie.202113235</a></p>
