<p class="2023">Yoo S, Garg E, Elliott LT, Hung RJ, Halevy AR, Brooks JD, Bull SB, Gagnon F, Greenwood C, Lawless JF, Paterson AD, Sun L, Zawati MH, Lerner-Ellis J, Abraham R, Birol I, Bourque G, Garant JM, Gosselin C, Li J, Whitney J, Thiruvahindrapuram B, Herbrick JA, Lorenti M, Reuter MS, Adeoye OO, Liu S, Allen U, Bernier FP, Biggs CM, Cheung AM, Cowan J, Herridge M, Maslove DM, Modi BP, Mooser V, Morris SK, Ostrowski M, Parekh RS, Pfeffer G, Suchowersky O, Taher J, Upton J, Warren RL, Yeung R, Aziz N, Turvey SE, Knoppers BM, Lathrop M, Jones S, Scherer SW, Strug LJ. HostSeq: a Canadian whole genome sequencing and clinical data resource. <a href=https://doi.org/10.1186/s12863-023-01128-3>BMC Genom Data. 2023 May 2;24(1):26. doi: 10.1186/s12863-023-01128-3. PMID: 37131148; PMCID: PMC10152008.</a></p>

<p class="2023">Coombe, L., Warren, R. L., Wong, J., Nikolic, V., Birol, I.  ntLink: A Toolkit for De Novo Genome Assembly Scaffolding and Mapping Using Long Reads. <a href=https://doi.org/10.1002/cpz1.733> Current Protocols, 3(4), e733 (2023). https://doi.org/10.1002/cpz1.733</a></p>

<p class="2023">Yang C., Lo T., Nip K.M., Hafezqorani S., Warren R.L., Birol I., Characterization and simulation of metagenomic nanopore sequencing data with Meta-NanoSim.<a href=https://doi.org/10.1093/gigascience/giad013> GigaScience, Volume 12, 2023, giad013, https://doi.org/10.1093/gigascience/giad013 PMID: 36939007; PMCID: PMC10025935</a></p>

<p class="2023">Li C., Warren R.L., Birol I. Models and data of AMPlify: a deep learning tool for antimicrobial peptide prediction. <a href=https://doi.org/10.1186/s13104-023-06279-1>BMC Res Notes 16, 11 (2023). https://doi.org/10.1186/s13104-023-06279-1</a></p>
