<p class="2023">Lo T, Coombe L, Gagalova KK, Marr A, Warren RL, Kirk H, Pandoh P, Zhao Y, Moore RA, Mungall AJ, Ritland C, Pavy N, Jones SJM, Bohlmann J, Bousquet J, Birol I, Thomson A. 2023. Assembly and annotation of the black spruce genome provide insights on spruce phylogeny and evolution of stress response. <a href=https://doi.org/10.1093/g3journal/jkad247>G3 Genes|Genomes|Genetics, jkad247, https://doi.org/10.1093/g3journal/jkad247</a></p>

<p class ="2023">Li JX, Fernandez KX, Ritland C, Jancsik S, Engelhardt DB, Coombe L, Warren RL, van Belkum MJ, Carroll AL, Vederas JC, Bohlmann J, Birol I. 2023. Genomic virulence features of Beauveria bassiana as a biocontrol agent for the mountain pine beetle population.<a href=https://doi.org/10.1186/s12864-023-09473-4> BMC Genomics 24, 390. https://doi.org/10.1186/s12864-023-09473-4</a></p>

<p class="2023">Allison MJ, Warren RL, Lopez ML, Acharya-Patel N, Imbery JJ, Coombe L, Yang CL, Birol I, Helbing CC. 2023. Enabling robust environmental DNA assay design with “unikseq” for the identification of taxon-specific regions within whole mitochondrial genomes. <a href=https://doi.org/10.1002/edn3.438>Environmental DNA, 00, 1-16. https://doi.org/10.1002/edn3.438</a></p>

<p class="2023">Nip KM, Hafezqorani S, Gagalova KK, Chiu R, Yang C, Warren RL, Birol I. 2023. Reference-free assembly of long-read transcriptome sequencing data with RNA-Bloom2. <a href=https://doi.org/10.1038/s41467-023-38553-y>Nat Commun 14, 2940. https://doi.org/10.1038/s41467-023-38553-y</a></p>

<p class="2023">Wong J, Coombe L, Nikolić V, Zhang E, Nip KM, Sidhu P, Warren RL, Birol I. 2023. Linear time complexity de novo long read genome assembly with GoldRush. <a href=https://doi.org/10.1038/s41467-023-38716-x>Nat Commun 14, 2906. https://doi.org/10.1038/s41467-023-38716-x</a></p>

<p class="2023">Yoo S, Garg E, Elliott LT, Hung RJ, Halevy AR, Brooks JD, Bull SB, Gagnon F, Greenwood C, Lawless JF, Paterson AD, Sun L, Zawati MH, Lerner-Ellis J, Abraham R, Birol I, Bourque G, Garant JM, Gosselin C, Li J, Whitney J, Thiruvahindrapuram B, Herbrick JA, Lorenti M, Reuter MS, Adeoye OO, Liu S, Allen U, Bernier FP, Biggs CM, Cheung AM, Cowan J, Herridge M, Maslove DM, Modi BP, Mooser V, Morris SK, Ostrowski M, Parekh RS, Pfeffer G, Suchowersky O, Taher J, Upton J, Warren RL, Yeung R, Aziz N, Turvey SE, Knoppers BM, Lathrop M, Jones S, Scherer SW, Strug LJ. 2023. HostSeq: a Canadian whole genome sequencing and clinical data resource. <a href=https://doi.org/10.1186/s12863-023-01128-3>BMC Genom Data. 24(1):26. doi: 10.1186/s12863-023-01128-3. PMID: 37131148; PMCID: PMC10152008.</a></p>

<p class="2023">Coombe, L., Warren, R. L., Wong, J., Nikolic, V., Birol, I. 2023. ntLink: A Toolkit for De Novo Genome Assembly Scaffolding and Mapping Using Long Reads. <a href=https://doi.org/10.1002/cpz1.733> Current Protocols, 3(4), e733. https://doi.org/10.1002/cpz1.733</a></p>

<p class="2023">Yang C., Lo T., Nip K.M., Hafezqorani S., Warren R.L., Birol I. 2023. Characterization and simulation of metagenomic nanopore sequencing data with Meta-NanoSim.<a href=https://doi.org/10.1093/gigascience/giad013> GigaScience, Volume 12, giad013, https://doi.org/10.1093/gigascience/giad013 PMID: 36939007; PMCID: PMC10025935</a></p>

<p class="2023">Li C., Warren R.L., Birol I. 2023. Models and data of AMPlify: a deep learning tool for antimicrobial peptide prediction. <a href=https://doi.org/10.1186/s13104-023-06279-1>BMC Res Notes 16, 11. https://doi.org/10.1186/s13104-023-06279-1</a></p>
