---  
layout: post  
title: "Manuscript describing unikseq and its application to species monitoring using environmental DNA (eDNA), published"  
category: news  
---  

In our study published in the [Environmental DNA journal](https://doi.org/10.1002/edn3.438) we present [unikseq](https://github.com/bcgsc/unikseq), a comparative genomics utility that uses words of length k (k-mers) to quickly identify unique regions in genome sequences, which can be used to yield highly specific quantitative real-time polymerase chain reaction (qPCR)-based [eDNA](https://en.wikipedia.org/wiki/Environmental_DNA) assays. In our manuscript, we illustrate its application within an animal mitochondrial genome (mitogenome) assay development workflow for the design and validation of eDNA assays respectively selective for American bullfrog, burbot, lake trout, and quillback rockfish. Unikseq scales linearly in execution time and compute memory usage, hereby facilitating [concurrent comparison of thousands to millions of mitogenomes, SARS-CoV-2 genomes, etc., and - with its Bloom filter version distributed with releases v1.3.1 and over - multiple large Gbp-size genomes](https://raw.githubusercontent.com/bcgsc/unikseq/main/unikseq-recomb2023poster.png) including those of human (3 billion nucleotides) and spruce (20 billion nucleotides). [Unikseq is freely available on github](https://github.com/bcgsc/unikseq).
